include/deal.II/lac/block_vector_base.h

```diff
@@ -723,6 +740,14 @@ class BlockVectorBase : public Subscriptor
   BlockVectorBase &
   operator= (const BlockVectorBase &V);
 
+#ifdef DEAL_II_WITH_CXX11
+  /**
+   * Move assignment operator. Move each block of the vector @p V into the
+   * current object if `VectorType` is move-constructible, otherwise copy them.
```

Comment:
Here, the markup actually makes sense. But I don't know whether doxygen understands mark-down-style backticks. Just write it as `<code>VectorType</code>` unless you know for sure that this will be rendered correctly.